opba-auth-rest-api/src/main/resources/static/tpp_auth_api.yml

```diff
@@ -159,7 +189,7 @@ components:
         Bad username or password
       headers:
         X-Request-ID:
-          $ref: '#/components/parameters/X-Request-ID'
+          $ref: "#/components/parameters/X-Request-ID"
```

Comment:
Double quotes process '\n' characters in string as if it would be new line, this is not what is desired in URL path references, please use `'` for quoting